tests/server.go

```diff
@@ -0,0 +1,49 @@
+package tests
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+func NewHTTPServerPod(port int) *corev1.Pod {
+	serverCommand := fmt.Sprintf("nc -klp %d --sh-exec 'echo -e \"HTTP/1.1 200 OK\\nContent-Length: 12\\n\\nHello World!\"'", port)
```

Comment:
Didn't we need a `while` for lifeness ?

Reply:
No, we have `-k` in the arguments, which will keep the server alive.

Reply:
It works, I've checked. I think you were missing passing the `-e` to `echo` .